### PR TITLE
Mm new langs

### PR DESCRIPTION
**Pull request by mXaln**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,8 +22,8 @@ import { default as TouchBackend } from 'react-dnd-touch-backend';
 import config from './config/config';
 
 // import and configure the raven client for sentry in order to track errors
-import Raven from 'raven-js';
-Raven.config(`http://9183fe1745da4049889061d44d154a4b@${config.domain}:9000/3`).install();
+//import Raven from 'raven-js';
```
Comment (natibekele) [inline src/App.js]: My only question is why are we removing sentry? I'm just curious

Comment (mXaln) [reply src/App.js]: oh good, catch @natibekele. I forgot to un-comment it. Fixed now.

File: src/App.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ class App extends Component {
 
   componentDidCatch(error, info) {
     this.setState({hasError: true});
-    Raven.captureException(error, {extra: info}); //send error to raven client
+    //Raven.captureException(error, {extra: info}); //send error to raven client
```
Comment (natibekele) [inline src/App.js]: it's also commented out here? i'm guessing similar thing here with just comment by mistake?

Comment (mXaln) [reply src/App.js]: :man_facepalming: Thanks, @natibekele !

**Pull request merged**